api/src/main/resources/workbench.yaml

```diff
@@ -3549,6 +3607,36 @@ definitions:
       notebookName:
         type: string
 
+  RecentWorkspaceRequest:
+    type: object
+    properties:
+      workspaceId:
+        type: integer
+        format: int64
+      userId:
+        type: integer
+        format: int64
+
+  RecentWorkspace:
+    type: object
+    properties:
+      workspaceId:
```

Comment:
FWIW, I would tend to leave off the prefix on the property names, since we know we're talking about a workspace. So just `id`, `namespace`, `firecloudName`, etc.